x/bank/client/cli/tx.go

```diff
@@ -8,14 +8,72 @@ import (
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/context"
 	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	clientx "github.com/cosmos/cosmos-sdk/client/tx"
```

Comment:
@alexanderbez this looks like a duplicate import, would you like me to remove this in slashing PR

Reply:
yes, please! good catch :)